Watchman.Discord/Areas/Users/Controllers/UsersController.cs

```diff
@@ -40,7 +40,7 @@ public void GetAvatar(SocketMessage message)
         [DiscordCommand("-add role")]
         public void AddRole(SocketMessage message)
         {
-            var commandRole = message.Content.Replace("-add role ", string.Empty);
+            var commandRole = message.Content.ToLower().Replace("-add role ", string.Empty);
```

Comment:
https://stackoverflow.com/questions/6225808/string-tolower-and-string-tolowerinvariant
```suggestion
            var commandRole = message.Content.ToLowerInvariant().Replace("-add role ", string.Empty);
```